### PR TITLE
merge 1.0.2 with all the new shiny features

### PR DESCRIPTION
**Pull request by vecna**
### Added
- New simple visualization based on isotope, for the personal page
### Fixed
- License in package.json is AGPL

### Added
- project status and vision
- new parser pipeline logic
- links about DataBeer and CCC talks
- license AGPL 3.0

### Status
- as described in the project status: https://github.com/tracking-exposed/presentation/raw/master/fbTREX%20-%20project%20status%20and%20analysis%20-%20December%202018.pdf

### CODE REVIEW

File: lib/echoes.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 // debugger module, suggested song: echoes pink floyd
 var elasticsearchClient = require("./elasticsearch")
+var debug = require('debug')('lib:echos');
```
Comment (joxer) [inline lib/echoes.js]: var debug = require('debug')('lib:echoes');

File: routes/events.js [hunk 2]
```diff
@@ -194,41 +204,32 @@ function promisifyInputs(body, geoinfo, supporter) {
...
     }
-    
-    debug(" * user %d [%s] last activity %s (%s ago) %s",
-        supporter.userId, geoinfo,
-        moment(supporter.lastActivity).format("HH:mm DD/MM"),
-        moment.duration(moment.utc()-moment(supporter.lastActivity)).humanize(),
-          supporter.version);
-    
+
     var ts = Math.round((new Date()).getTime() / 1000);
```
Comment (joxer) [inline routes/events.js]: @vecna let's think if we should put a milliseconds version or a random UUID version here

File: routes/events.js [hunk 3]
```diff
@@ -194,41 +204,32 @@ function promisifyInputs(body, geoinfo, supporter) {
...
-    
-*/
+        id: ts,
+        index: "fbtrex_users",
+        pseudo: supporter.userId,
+        geo: geoinfo,
+        last_activity: supporter.lastActivity,
+        timelines: processed_timelines,
+        impressions: processed_impressions,
+        htmls: processed_html
```
Comment (joxer) [inline routes/events.js]: maybe we can add a timestamp here to log when this data was sent?

File: lib/echoes.js (hunk 1 above)
Comment (vecna) [reply lib/echoes.js]: ok

File: support/elasticsearch/summary.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+    "mappings" : {
+        "doc" : {
+            "dynamic": false,
+            "properties": {
+                "id": { "type": "integer" },
+                "errors": { "type": ??? List ??? },
```
Comment (vecna) [inline support/elasticsearch/summary.json]: @joxer what should be put here?

**Pull request merged**